src/EditorFeatures/Core/Implementation/RenameTracking/RenameTrackingTaggerProvider.TrackingSession.cs

```diff
@@ -194,7 +194,7 @@ public void Cancel()
                     // Get the source symbol if possible
                     var sourceSymbol = await SymbolFinder.FindSourceDefinitionAsync(symbol, document.Project.Solution, _cancellationToken).ConfigureAwait(false) ?? symbol;
 
-                    if (!sourceSymbol.Locations.All(loc => loc.IsInSource))
+                    if (!sourceSymbol.Locations.Any() || !sourceSymbol.Locations.All(loc => loc.IsInSource))
```

Comment:
The symbol is neither from source nor metadata? @CyrusNajmabadi has convinced me this isn't insane, but I'm wondering if this needs to be doc'ed somewhere.